charters/charter-2021.html

```diff
@@ -0,0 +1,841 @@
+            <dd>The Web of Things Interest Group and Working Group discuss the architecture of the Web of Things. This architecture may prove useful to model some of the features investigated by the Media and Entertainment Interest Group such as cloud browsers.</dd>
+          </dl>
+
+          <h3 id="external-coordination">External Organizations</h3>
+          <p>There are a number of external groups working in areas related to the ones
+in scope for the Media and Entertainment IG. The Interest Group should determine whom to
+communicate with and then maintain communication with them. The following
+groups are likely to be important: </p>
+          <dl>
+            <dt><a href="https://aomedia.org/">Alliance for Open Media</a></dt>
+            <dd>The Alliance for Open Media is founded by leading Internet
+            companies focused on developing next-generation media formats,
+            codecs and technologies.</dd>
+
+            <dt><a href="https://www.ansi.org/">ANSI</a></dt>
+            <dd>The American National Standards Institute is a private, non-profit
+                organization that oversees the development of voluntary consensus
+                standards for products, services, processes, systems, and personnel in
+                the US.</dd>
+
+            <dt><a href="https://www.arib.or.jp/english/">ARIB</a></dt>
+            <dd>The Association of Radio Industries and Businesses is aimed to conduct
+                investigation, research &amp; development and consultation of utilization
+                of radio waves from the view of developing radio industries, and to
+                promote realization and popularization of new radio systems in the field
+                of telecommunications and broadcasting.</dd>
+
+            <dt><a href="https://www.atsc.org/cms/">ATSC</a></dt>
+            <dd>The Advanced Television Systems Committee provides standards for
+                digital television transmission over terrestrial, cable, and satellite
+                networks.</dd>
+
+            <dt><a href="https://www.cablelabs.com/">CableLabs</a></dt>
+            <dd>Cable Television Laboratories, Inc. is a non-profit research and
+                development consortium that has cable operators as its members, and works
+                for requirements for new technologies and new services.</dd>
+
+            <dt><a href="https://www.cta.tech/">CTA</a></dt>
+            <dd>The Consumer Technology Association is the trade
+                organization for the consumer electronics industry,
+                which promotes the consumer technology industry
+                through technology policy, events, research, promotion
+                and the fostering of business and strategic
+                relationships. CTA organizes the annual Consumer
+                Electronics Show (CES), and hosts the 
+                <a href="https://standards.cta.tech/kwspub/wave/">CTA
```

Comment:
One more WAVE URL update
```suggestion
                <a href="https://cta.tech/WAVE">CTA
```